docs/ena_readme.md

```diff
@@ -0,0 +1,72 @@
+## Run GGP using ENA data
+
+ENA is very useful resource for MAGs/bins generation because it [structured](https://www.ebi.ac.uk/training/online/courses/ena-quick-tour/what-is-ena/how-is-ena-structured/) raw-reads and retrieved assemblies in study format.
+
+For example, 
+`raw-reads study` [SRP080008](https://www.ebi.ac.uk/ena/browser/view/PRJNA330077) was assembled and submitted as `assembly study` [ERP108081](https://www.ebi.ac.uk/ena/browser/view/PRJEB26108).
+
+It is possible to submit generated MAGs to ENA **if you own the data**. That pipeline is automated to register and submit MAGs using [genome_uploader](https://github.com/EBI-Metagenomics/genome_uploader) to ENA mags layer under `assembly study` project. 
+> [!NOTE]
+> If you do not have full access to ENA studies - do not submit MAGs 
+
+## Simple run without MAGs upload
+```bash
+nextflow run ebi-metagenomics/genomes-generation \
+-profile `specify profile(s)` \
+--ena_assembly_study_accession `assembly study` \
+--ena_raw_reads_study_accession `raw-reads study` \
+--outdir `full path to output directory`
+```
+
+## Run with MAGs upload
+```bash
+nextflow run ebi-metagenomics/genomes-generation \
+-profile `specify profile(s)` \
+--ena_assembly_study_accession `assembly study` \
+--ena_raw_reads_study_accession `raw-reads study` \
+--metagenome "chosen metagenome" \
+--biomes "chosen biome,chosen feature,chosen material" \ 
+--centre_name "name" \
+--upload_tpa true \
```

Comment:
No need to set them to `true`, from `https://www.nextflow.io/docs/latest/cli.html#pipeline-parameters`.

```
Parameters that are specified on the command line without a value are set to true.
```